benchmarks/distributed_tree_driver/distributed_tree_driver.cpp

```diff
@@ -285,22 +291,26 @@ int main_(std::vector<std::string> const &args, const MPI_Comm comm)
               << "#queries/MPI process    : " << n_queries << '\n'
               << "size of shift           : " << shift << '\n'
               << "dimension               : " << partition_dim << '\n'
+              << "shift-queries           : " << shift_queries << '\n'
               << '\n';
   }
 
   Kokkos::View<ArborX::Point *, DeviceType> random_points("random_points", 0);
+  Kokkos::View<ArborX::Point *, DeviceType> random_queries("random_queries", 0);
   {
-    // Random points are "reused" between building the tree and performing
-    // queries. Note that this means that for the points in the middle of
-    // the local domains there won't be any communication.
-    auto n = std::max(n_values, n_queries);
-    Kokkos::resize(random_points, n);
+    // By default, random points are "reused" between building the tree and
+    // performing queries. Note that this means that for the points in the
+    // middle of the local domains there won't be any communication.
```

Comment:
Without this patch, the locations related to the individual queries coincide with the points (or boxes) we construct the BVH with. Hence, there is no communication required for queries inside the local boxes. This is different after this patch because we typically need to forward most of the queries.

Reply:
I don't understand what it means middle here. And won't it depend on the original subdomain shift, too? If you have shift > 0.5, all points will be communicated.

Reply:
Sure, but the default is `shift=1`. If all that is too confusing, I am happy to just remove the comment or replace it with something better if you have a suggestion.